addOns/llm/src/main/java/org/zaproxy/addon/llm/services/LlmCommunicationService.java

```diff
@@ -207,7 +214,7 @@ public void reviewAlert(Alert alert) {
     }
 
     private static boolean isPreviouslyReviewed(Alert alert) {
-        return !alert.getTags().containsKey(AI_REVIEWED_TAG_KEY);
+        return alert.getTags().containsKey(AI_REVIEWED_TAG_KEY);
```

Comment:
This should be in its own PR with proper tests so we don't end up breaking this again with refactorings.

Reply:
The new changes should have tests too.

Reply:
I'll create another PR.

Reply:
https://github.com/zaproxy/zap-extensions/pull/6651